app/controllers/admin/pages_controller.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: UTF-8
+class Admin::PagesController < ApplicationController
+  load_permissions_and_authorize_resource find_by: :url
+  before_action :authorize, except: [:edit, :update]
+
+  def index
+    @page_grid = initialize_grid(Page, include: 'council')
+  end
+
+  def new
+    @page = Page.new
+  end
+
+  def edit
+    @page = Page.find_by!(url: params[:id])
+    authorize!(:manage, @page)
+  end
+
+  def create
+    @page = Page.new(page_params)
+    if @page.save
+      redirect_to edit_admin_page_path(@page), notice: alert_create(Page)
+    else
+      render :new, status: 422
+    end
+  end
+
+  def update
+    @page = Page.find_by!(url: params[:id])
+    authorize!(:manage, @page)
```

Comment:
Makes sense 😄 

Hmm, how would it be easiest to implement the ability for a certain user to edit a page?
Create a `PageUser` model and assign users to a page?
Some pages will only be edited by some posts, and some temporary pages only by some people.